api/features/workflows/core/models.py

```diff
@@ -84,7 +84,9 @@ def commit(self, committed_by: "FFAdminUser"):
         self.save()
 
     def is_approved(self):
-        return self.environment.minimum_change_request_approvals is not None and (
+        if self.environment.minimum_change_request_approvals is None:
```

Comment:
Feels like this might be slightly more pythonic written as: 

```python
return self.environment.minimum_change_request_approvals is None or (
    self.approvals.filter(approved_at__isnull=False).count()
    >= self.environment.minimum_change_request_approvals
)
```